.changeset/changeset-nightly.md

```diff
@@ -0,0 +1,6 @@
+---
+"@near-wallet-selector/nightly": major
+---
```

Comment:
this change should be a patch instead, based on semantic versioning. could you update it?

Reply:
done